packages/app-desktop/plugins/GotoAnything.tsx

```diff
@@ -378,6 +379,7 @@ class Dialog extends React.PureComponent<Props, State> {
 		});
 
 		if (item.type === BaseModel.TYPE_COMMAND) {
+			focusEditorIfEditorCommand(item.id, CommandService.instance());
```

Comment:
I think this line should come after the command is executed. It's just a small thing that likely doesn't matter, but it is best to keep the same order in both scripts to prevent inconsistency for the user.